mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -150,7 +150,10 @@ func TestClientSideEncryptionProse(t *testing.T) {
 		alteredKeydoc, _ = bsoncore.AppendDocumentEnd(alteredKeydoc, cidx)
 
 		// Insert the copied key document into keyvault.datakeys with majority write concern.
-		wcMajority := writeconcern.New(writeconcern.WMajority(), writeconcern.WTimeout(1*time.Second))
+		wcMajority := &writeconcern.WriteConcern{
```

Comment:
Can we use the convenience function in the example, instead of using the "majority" magic string? 

```go
wcMajority := writeconcern.Majority()
wcMajority.WTimeout = 1 * time.Second
```